ulogd: init at 2.0.7

xvuko opened the pull request:
###### Motivation for this change
> ulogd is a userspace logging daemon for netfilter/iptables related logging. This includes per-packet logging of security violations, per-packet logging for accounting, per-flow logging and flexible user-defined accounting. 

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

xvuko committed:
maintainers: add xvuko

xvuko committed:
ulogd: init at 2.0.7

xvuko committed:
nixos/ulogd: init

aanderse reviewed (dismissed):
I took a look at your module and had a few suggestions. I hope you find them helpful. If you have any questions or need any help don't hesitate to ask.

xvuko committed:
linuxdoc-tools: init at 0.9.82

xvuko commented:
@aanderse Thanks for suggestions! I have added them to my commits.

In order to build docs I added another package: linuxdoc-tools. Is it OK to add it here, or should I open another PR?

xvuko commented on nixos/modules/services/logging/ulogd.nix:
```diff
@@ -0,0 +1,53 @@
+        '';
+        type = types.lines;
+        default = "";
+        description = "Ulogd config.";
```
There is no up-to-date documentation available online so I added information about docs in ${pkgs.ulogd.doc}/share/ulogd/.

aanderse reviewed (commented):
I'm not familiar with `ulogd` and I don't generally review packages, but the **module** part of this PR looks good to me :+1:

alexvorobiev commented:
@xvuko I am interested in linuxdoc-tools. Did you consider separating it to another PR?

RaitoBezarius commented:
Done in 207453
